test/functional/rpc_setban.py

```diff
@@ -70,6 +71,11 @@ def run_test(self):
         assert not self.is_banned(node, tor_addr)
         assert not self.is_banned(node, ip_addr)
 
+        self.log.info("Test -bantime")
+        self.restart_node(1, ["-bantime=1234"])
+        self.nodes[1].setban("127.0.0.1", "add")
+        banned = self.nodes[1].listbanned()[0]
+        assert_equal(banned['ban_duration'], 1234)
```

Comment:
The assert_equal function should be called with the expected value as the first argument and the actual value as the second argument.

```suggestion
        assert_equal(1234, banned['ban_duration'])
```

Reply:
It is generally a good idea to include a descriptive error message in the call to assert_equal so that it is clear what the error is if the assertion fails.

```suggestion
        assert_equal(1234, banned['ban_duration'], "ban_duration is not equal to 1234")
```

Reply:
The assertion will already print this when hit

Reply:
> The assertion will already print this when hit

Yes, there is no need to include a message.